releases/v0.2.0.md

```diff
@@ -0,0 +1,71 @@
+# Release Notes for v0.2.0
+Confidential Containers has adopted a six-week release cadence.
+This is our first release on this schedule.
+This release mainly features incremental improvements to our build system and tests
+as well as minor features, adjustments, and cleanup.
+
+Please see the [quickstart guide](../quickstart.md) for details on how to try out Confidential Containers
+
+## What's new
+
+- Kata CI uses existing Kata tooling to build components
+- Kata CI caches build environments for components
+- Incremental advances in signature support
```

Comment:
I wasn't really sure what the status is. I'd be happy for someone in the know to add a separate entry for it.

Reply:
@Xynnn007 - do you have anything, you think we should add here on cosign? Thanks

Reply:
Yea, thanks, now 
```
- Support cosign-signed image verification
```

Reply:
Ok, since it sounds like there are still some limitations I added the cosign support to the existing bullet about signature improvements. It will be great to have more documentation about cosign and signatures in general. I don't think many people know how to use it yet.